noxfile.py

```diff
@@ -518,7 +518,9 @@ def prerelease(session: nox.sessions.Session, tests_path):
         "--prefer-binary",
         "--pre",
         "--upgrade",
-        "pandas",
+        # TODO(shobs): Remove tying to version 2.1.3 after
+        # https://github.com/pandas-dev/pandas/issues/56463 is resolved
+        "pandas==2.1.3",
```

Comment:
Another possibility is to exclude the current version using `pandas!=2.1.4` so that we can run pre-release tests against the next release. It will be toilsome to keep updating the version but the process of updating the excluded versions could be used as a way to follow up on the issue.

Reply:
Thanks @parthea for the suggestion. Done so.